src/auro-sidenav.js

```diff
@@ -138,12 +144,52 @@ export class AuroSidenav extends LitElement {
     this.addEventListener('mousedown', this.handleMouseDown);
   }
 
+  async updated(changedProperties) {
+    super.updated(changedProperties);
+    await this.updateComplete;
+
+    if (!this.static) {
+      const sidenavMobileAccordion = this.shadowRoot.getElementById("accordion");
+      if (sidenavMobileAccordion) {
+        // #accordionContent is the farthest down we can drill in the DOM to still catch all nested accordion events
+        const sidenavMobileAccordionContent = sidenavMobileAccordion.shadowRoot.querySelector('div.componentWrapper > #accordionContent');
+
+        // Catch all nested accordion expansion events
+        sidenavMobileAccordionContent.addEventListener("toggleExpanded", (event) => {
+          const nestedAccordionElement = event.target;
+          const nestedAccordionContent = nestedAccordionElement.shadowRoot.querySelector('div.componentWrapper > #accordionContent');
+
+          // Set height to auto to allow expansion/contraction
+          sidenavMobileAccordionContent.style.height = "auto";
+
+          const onTransitionEnd = () => {
+            // Transition is over, now update sidenav "root" accordion with correct height
+            sidenavMobileAccordion.handleContentSlotChanges();
+            // Make listener clean itself up
+            nestedAccordionContent.removeEventListener("transitionend", onTransitionEnd);
+          };
+
+          nestedAccordionContent.addEventListener("transitionend", onTransitionEnd);
+        });
+      }
+    }
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
-    return html`
+    const sidebarContent = html`
```

Comment:
Add a callback so that when the DOM rendered this element, `role="nav"` is added. Maybe... 

```js
  connectedCallback() {
    super.connectedCallback();
    this.setAttribute('role', 'nav');
  }
```